ocfweb/docs/docs/services/hpc.md

```diff
@@ -0,0 +1,65 @@
+[[!meta title="High Performance Computing"]]
+
+**NOTE: We are in the process of trialling this service to users so that we can make the service as accommodating and secure as possible. This means that items concerning the service, including this documentation, are subject to change. We will do our best to keep everyone updated and notified of changes as they come.**
+
+## Introduction
+
+At the OCF we offer a High Performance Computing (HPC) service for individuals and groups that need to run computationally demanding software. Our main HPC server is currently named `corruption`, however we also have plans to expand the cluster to maximize the use of the resources at our disposal.
+
+### Gaining Access
+
+In order to access the HPC cluster, you must send a proposal to [help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu). Make sure to include your OCF username or group account name and a short, but detailed technical description of the experiments you plan to run on our HPC infrastructure. This would include information about the nature of the software being developed, as well as the amount of computational resources that are expected to be needed.
+
+#### Connecting
+
+Our Slurm master node is currently `segfault`. Once you submit your proposal and are approved access you will be able to connect via ssh by running the following command:
+
+    ssh my_ocf_username@segfault.ocf.berkeley.edu
+
+If you have trouble connecting please email us at [help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu).
+
+### The Cluster
+
+Users currently have access to our main HPC server `corruption`, which has the following specifications:
+
+* 2x NVIDIA GTX 1080 Ti
+* 1x Intel Intel(R) Xeon(R) CPU E5-2640 v4
+* 4x 16Gb ECC DDR4-2400 ram
+
+We have also recently been approved a grant that will be awarded in the coming weeks. So in addition to the above, the following will be installed in `corruption`:
```

Comment:
"... will [additionally] be installed into `corruption`"

If we are 100% being "awarded" this grant in a couple weeks, might as well include that in the specs section? I don't think anyone is requesting HPC jobs over the summer

Reply:
Realistically I think this be longer than a couple weeks, perhaps before the end of the summer? Hard to say about timing on these grants though, probably better not to bet on them :P

Reply:
The STF email stated mid July so we have a bit to wait.